services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -166,6 +166,11 @@ def project_summary_lead_party_guid(self):
         if self.project.project_lead_party_guid:
             return self.project.project_lead_party_guid
         return None
+    
+    @hybrid_property
+    def project_lead_email(self):
```

Comment:
```suggestion
    def project_lead_email(self) -> str | None:
```
I think is how python 3.12 does optional params now, using the logical OR operator `|`.